oak_channel/src/message.rs

```diff
@@ -30,6 +30,8 @@ static_assertions::assert_eq_size!([u8; INVOCATION_ID_SIZE], InvocationId);
 
 pub const BODY_OFFSET: usize = 8;
 
+// messages never have a length of zero
```

Comment:
Do they not? (anyways I agree with disabling the lint)

Reply:
The `.len()` always includes a fixed size header. Technically they could be empty in so far as that they have a body of `0`, but the clippy rule expects a `is_empty()` to be true for `len() === 0`. 

Updated the comment for clarity.